common/proto/common/SDMS_Auth.proto

```diff
@@ -116,6 +121,8 @@ message UserSetAccessTokenRequest
     required string             access      = 1; // Access token
     required uint32             expires_in  = 2; // Access expiration time in seconds
     required string             refresh     = 3; // Refresh token (never expires)
+    optional AccessTokenType    type        = 4;
```

Comment:
Would this mean that optional params can be added as part of the object in the request here?

https://github.com/ORNL/DataFed/blob/devel/web/datafed-ws.js#L1529

Reply:
Yes, exactly!